src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -1157,7 +1157,7 @@ class DrawSketchHandlerDimension : public DrawSketchHandler
         , selLine({})
         , selCircleArc({})
         , selEllipseAndCo({})
-        , initialSelection(SubNames)
+        , initialSelection(std::move(SubNames))
```

Comment:
This change without the mandatory change of the argument list of the constructor is dangerous as it destroys the passed string of the calling instance.
The change is: `DrawSketchHandlerDimension(std::vector<std::string> SubNames)`, i.e. the ampersand must be removed.

Reply:
Ah sorry about that. I just forced push